controllers/admin/AdminDashboardController.php

```diff
@@ -425,7 +425,7 @@ public function ajaxProcessSaveDashConfig()
         $hook = Tools::getValue('hook');
         $configs = Tools::getValue('configs');
 
-        if (!in_array($hook, self::DASHBOARD_ALLOWED_HOOKS)) {
+        if (!in_array(str_replace('hook', '', $hook), self::DASHBOARD_ALLOWED_HOOKS)) {
```

Comment:
```suggestion
        if (!in_array(lcfirst(str_replace('hook', '', $hook)), self::DASHBOARD_ALLOWED_HOOKS)) {
```
When trying locally, I found you also need to apply lcfirst